src/features/instance/applications/new/ImportProjectForm.tsx

```diff
@@ -39,12 +44,45 @@ export function ImportProjectForm() {
 		},
 	});
 
+	const restartingInstance = () => {
+		const toastId = toast.loading('Restarting', {
+			description: `Restarting ${targetNoun.toLowerCase()}. This may take up to 60 seconds.`,
```

Comment:
Our replicated restart actually returns a job id, so the UI can monitor the progress of the restart and let the user know when it is finished (rather than just generically saying it will be a minute).
Not something we need for v1 though.